### ISSUE TITLE
Add captcha.io service provider

### ISSUE THREAD

**Issue by peter279k** [lorien/captcha_solver#19]
If possible, we can consider adding `captchas.io` service.

Once this issue is accepted, I can help this issue.

### References
- official website: https://captchas.io
- API documentation: https://api.captchas.io/document

**Comment by captchasio** [comment]
Yes it would be nice... ask me for information about integrating our service to your application.
